### PR TITLE
feat(react-property): create package for HTML and SVG DOM property configs

### PR DESCRIPTION
**Pull request by remarkablemark**
Relates to remarkablemark/html-react-parser#107

Would love to get your review @talbet

### CODE REVIEW

File: packages/react-property/scripts/build-html.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ */
+DOMProperty.injection.injectDOMPropertyConfig(HTMLDOMPropertyConfig);
+
+/**
+ * Create output directories (if it does not exist).
+ */
+try {
+  fs.mkdirSync(LIB_DIR);
+} catch (error) {
+  // throw error;
```
Comment (talbet) [inline packages/react-property/scripts/build-html.js]: Should the error be re-thrown here if it is not a "directory already exists" error? 
```
if (error.code !== 'EEXIST') throw error;
```

File: packages/react-property/scripts/build-html.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+try {
+  fs.mkdirSync(HTML_DIR);
+} catch (error) {
+  // throw error;
+}
+
+/**
+ * HTML DOM property config.
```
Comment (talbet) [inline packages/react-property/scripts/build-html.js]: Maybe we can add extra context to this comment? I wasn't sure what it was for at first. Something along the lines of: "an object containing HTML attributes and how they should be mapped to React props"

File: packages/react-property/scripts/build-svg.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// `autofocus` is removed since it's not an applicable attribute for SVG elements
+delete DOMProperty.getPossibleStandardName.autofocus;
+
+/**
+ * Create output directories (if it does not exist).
+ */
+try {
+  fs.mkdirSync(LIB_DIR);
+} catch (error) {
+  // throw error;
```
Comment (talbet) [inline packages/react-property/scripts/build-svg.js]: as above, we could handle this error if it is not expected
```
if (error.code !== 'EEXIST') throw error;
```

File: packages/react-property/src/html/index.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+var attributeToProperty = require('../../lib/html/attribute-to-property');
+var attributes = require('../../lib/html/attributes');
+var booleanProperties = require('../../lib/html/boolean-properties');
+var overloadedBooleanProperties = require('../../lib/html/overloaded-boolean-properties');
+
+var attributeMap = {};
+var attributeName;
```
Comment (talbet) [inline packages/react-property/src/html/index.js]: Nice 👍 I haven't used `var` for such a long time I probably would have forgotten the scoping complications of `var` and loops.

File: packages/react-property/scripts/build-html.js (hunk 1 above)
Comment (remarkablemark) [reply packages/react-property/scripts/build-html.js]: 1566b2f

File: packages/react-property/scripts/build-svg.js (hunk 3 above)
Comment (remarkablemark) [reply packages/react-property/scripts/build-svg.js]: 1566b2f

File: packages/react-property/scripts/build-html.js (hunk 2 above)
Comment (remarkablemark) [reply packages/react-property/scripts/build-html.js]: 42999e2

I changed the wording slightly. Let me know if it makes sense.

File: packages/react-property/src/html/index.js (hunk 4 above)
Comment (remarkablemark) [reply packages/react-property/src/html/index.js]: Thanks. I'm considering rewriting this to TypeScript but that can be an enhancement done later.

File: packages/react-property/scripts/build-html.js (hunk 2 above)
Comment (talbet) [reply packages/react-property/scripts/build-html.js]: Yeah, I think this makes it clearer.

**Comment by remarkablemark** [comment]
Released and published [`react-property@0.1.0`](https://github.com/remarkablemark/react-dom-core/tree/react-property%400.1.0):

```sh
# npm
npm i -S react-property@0.1.0

# yarn
yarn add react-property@0.1.0
```

**Pull request merged**